### ISSUE TITLE
CNS axonal spheroid

### ISSUE THREAD

**Issue by pnrobinson** [obophenotype/human-phenotype-ontology#7416]
**Preferred term label:**
CNS axonal spheroid

**Synonyms**
Central nervous system axonal spheroid

**Definition (free text, please give PubMed ID)**
Spheroid is the general term for pathological swelling of neuronal processes, particularly axons, which is also referred to as axonal swelling. With H&E staining, spheroids are eosinophilic, and with silver staining such as Bodian staining, they are more clearly visualized.

TODO -- define relation to axonal swelling

**Parent term (use hpo.jax.org/app)**

**Diseases characterized by this term ? (e.g. Orphanet or OMIM number)**
PMID:27423618   OMIM: 221820 

**Your nano-attribution (ORCID)**

**Comment by pnrobinson** [comment]
Axonal spheroids are bubble-like biological features that form on most degenerating axons. During neurodegeneration, bubble-like swellings form along the length of the axon, a primary and early effect of dynamic axonal deformation. These “varicosities”, “spheroids”, or “swellings” have been described as beads on a string and branches bearing fruit, and they frequently appear on degenerating axons. The focal swellings on degenerating axons, axonal spheroids, are often filled with cellular debris such as organelles, pathological proteins, and disorganized cytoskeletal elements. 
PMID:34678457

**Comment by pnrobinson** [comment]
added
